packages/daemon/test/test-endo.js

```diff
@@ -166,10 +167,15 @@ test('cannot spawn worker with existing non-worker name', async t => {
   );
   const bootstrap = getBootstrap();
   const host = E(bootstrap).host();
-  const ten = await E(host).evaluate('MAIN', '10', [], [], 'ten');
-  t.is(ten, 10);
-  await t.throwsAsync(() => E(host).provideWorker('ten'), {
-    message: 'Not a worker "ten"',
+  const foo = await E(host).evaluate('MAIN', '10', [], [], 'foo');
+  t.is(foo, 10);
+
+  // This resolves with the existing value of 'foo' rather than overwriting it
+  // with a new worker.
+  await E(host).provideWorker('foo');
+  await t.throwsAsync(() => E(host).evaluate('foo', '20', [], [], 'bar'), {
+    message:
+      'Cannot deliver "evaluate" to target; typeof target is "undefined"',
```

Comment:
This is almost certainly a symptom of there being no entry in the worker weak map, or the internal facet for this formula being undefined anyway. I regret the internal/external separation and look forward to eliminating it.